From a74fb1bb4b149c96d8da731cb10a6437b0ce5b4d Mon Sep 17 00:00:00 2001
From: Antoine Gourlay <antoine@gourlay.fr>
Date: Thu, 29 Dec 2022 14:30:33 +0100
Subject: [PATCH] no git

---
 project/modules/shared.sc | 24 +++---------------------
 1 file changed, 3 insertions(+), 21 deletions(-)

diff --git a/project/modules/shared.sc b/project/modules/shared.sc
index 008d219..d1e1d6e 100644
--- a/project/modules/shared.sc
+++ b/project/modules/shared.sc
@@ -10,29 +10,11 @@ trait CsMima extends Mima {
 }
 
 def commitHash = T {
-  os.proc("git", "rev-parse", "HEAD").call().out.text.trim
+  "5ce0a0330f3ebc603f1d523b7886ef9b5425cb09"
 }
 
-lazy val latestTaggedVersion = os.proc("git", "describe", "--abbrev=0", "--tags", "--match", "v*")
-  .call().out
-  .trim
-lazy val buildVersion = {
-  val gitHead = os.proc("git", "rev-parse", "HEAD").call().out.trim
-  val maybeExactTag = scala.util.Try {
-    os.proc("git", "describe", "--exact-match", "--tags", "--always", gitHead)
-      .call().out
-      .trim
-      .stripPrefix("v")
-  }
-  maybeExactTag.toOption.getOrElse {
-    val commitsSinceTaggedVersion =
-      os.proc('git, "rev-list", gitHead, "--not", latestTaggedVersion, "--count")
-        .call().out.trim
-        .toInt
-    val gitHash = os.proc("git", "rev-parse", "--short", "HEAD").call().out.trim
-    s"${latestTaggedVersion.stripPrefix("v")}-$commitsSinceTaggedVersion-$gitHash-SNAPSHOT"
-  }
-}
+lazy val latestTaggedVersion = "v2.1.0-RC4"
+lazy val buildVersion = latestTaggedVersion.stripPrefix("v")
 
 trait PublishLocalNoFluff extends PublishModule {
   def emptyZip = T {
-- 
2.38.1

