From 3d2c93625bc025878681ee92d3f5d07fd8e52a7a Mon Sep 17 00:00:00 2001
From: Antoine Gourlay <antoine@gourlay.fr>
Date: Mon, 7 Feb 2022 22:48:47 +0100
Subject: [PATCH] no git

---
 project/modules/shared.sc | 24 +++---------------------
 1 file changed, 3 insertions(+), 21 deletions(-)

diff --git a/project/modules/shared.sc b/project/modules/shared.sc
index 51a27d0..b3c10f4 100644
--- a/project/modules/shared.sc
+++ b/project/modules/shared.sc
@@ -8,29 +8,11 @@ trait CsMima extends Mima {
 }
 
 def commitHash = T {
-  os.proc("git", "rev-parse", "HEAD").call().out.text.trim
+  "2868899e8296d46df2ccb3851d742c1cfb750ca1"
 }
 
-lazy val latestTaggedVersion = os.proc("git", "describe", "--abbrev=0", "--tags", "--match", "v*")
-  .call().out
-  .trim
-lazy val buildVersion = {
-  val gitHead = os.proc("git", "rev-parse", "HEAD").call().out.trim
-  val maybeExactTag = scala.util.Try {
-    os.proc("git", "describe", "--exact-match", "--tags", "--always", gitHead)
-      .call().out
-      .trim
-      .stripPrefix("v")
-  }
-  maybeExactTag.toOption.getOrElse {
-    val commitsSinceTaggedVersion =
-      os.proc('git, "rev-list", gitHead, "--not", latestTaggedVersion, "--count")
-        .call().out.trim
-        .toInt
-    val gitHash = os.proc("git", "rev-parse", "--short", "HEAD").call().out.trim
-    s"${latestTaggedVersion.stripPrefix("v")}-$commitsSinceTaggedVersion-$gitHash-SNAPSHOT"
-  }
-}
+lazy val latestTaggedVersion = "v2.1.0-RC2"
+lazy val buildVersion = latestTaggedVersion.stripPrefix("v")
 
 trait CoursierPublishModule extends PublishModule with JavaModule {
   import mill.scalalib.publish._
-- 
2.36.1

