From 88f902ca236237fc2001924289376f297770f3fd Mon Sep 17 00:00:00 2001
From: Antoine Gourlay <antoine@gourlay.fr>
Date: Sun, 9 May 2021 17:17:46 +0200
Subject: [PATCH 201/201] stick mima to fixed version for 2.0.x

---
 project/Mima.scala | 34 +---------------------------------
 1 file changed, 1 insertion(+), 33 deletions(-)

diff --git a/project/Mima.scala b/project/Mima.scala
index 3fdb03b..8382e1e 100644
--- a/project/Mima.scala
+++ b/project/Mima.scala
@@ -13,39 +13,7 @@ object Mima {
       .replace("-RC", "-")
       .forall(c => c == '.' || c == '-' || c.isDigit)
 
-  def binaryCompatibilityVersions: Set[String] = {
-
-    val latest = Seq("git", "describe", "--tags", "--abbrev=0", "--match", "v*")
-      .!!
-      .trim
-      .stripPrefix("v")
-
-    assert(latest.nonEmpty, "Could not find latest version")
-
-    if (stable(latest)) {
-      val prefix = latest.split('.').take(2).map(_ + ".").mkString
-
-      val head = Seq("git", "tag", "--list", "v" + prefix + "*", "--contains", "HEAD")
-        .!!
-        .linesIterator
-        .map(_.trim.stripPrefix("v"))
-        .filter(stable)
-        .toSet
-
-      val previous = Seq("git", "tag", "--list", "v" + prefix + "*", "--contains", "9c27d22ae")
-        .!!
-        .linesIterator
-        .map(_.trim.stripPrefix("v"))
-        .filter(stable)
-        .toSet
-
-      assert(previous.contains(latest) || latest == "2.0.0-RC6-7", "Something went wrong")
-
-      previous -- head
-    } else
-      Set()
-  }
-
+  def binaryCompatibilityVersions: Set[String] = Set("2.0.0-RC6-7")
 
   lazy val previousArtifacts = Seq(
     mimaPreviousArtifacts := {
-- 
2.31.1

