From 53f6aa8cf5a2da627601b348834cb8e7e40a000d Mon Sep 17 00:00:00 2001
From: Antoine Gourlay <antoine@gourlay.fr>
Date: Mon, 6 Dec 2021 11:53:38 +0100
Subject: [PATCH] build with unpackaged dependencies

---
 CMakeLists.txt | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c2f4341..bba7b88 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -10,8 +10,10 @@ option(LIBTREE_BUILD_TESTS "Build the tests" OFF)
 
 # Find all dependencies
 find_package(cxxopts CONFIG REQUIRED)
-find_package(elfio CONFIG REQUIRED)
-find_package(termcolor CONFIG REQUIRED)
+find_path(elfio elfio/elfio.hpp REQUIRED)
+
+# termcolor is unpackaged
+add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/bundled/termcolor ${CMAKE_CURRENT_BINARY_DIR}/termcolor EXCLUDE_FROM_ALL)
 
 include(CheckIncludeFiles)
 check_include_files("sys/auxv.h" LIBTREE_HAS_AUXV_HEADER)
@@ -28,8 +30,8 @@ set(SOURCES
 # Add the library
 add_library(libtree_lib OBJECT ${SOURCES})
 set_target_properties(libtree_lib PROPERTIES OUTPUT_NAME libtree)
-target_include_directories(libtree_lib PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include ${PROJECT_BINARY_DIR})
-target_link_libraries(libtree_lib termcolor::termcolor elfio::elfio)
+target_include_directories(libtree_lib PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include ${PROJECT_BINARY_DIR} ${elfio})
+target_link_libraries(libtree_lib termcolor::termcolor)
 
 # And the main cli tool
 add_executable(libtree ${CMAKE_CURRENT_SOURCE_DIR}/src/main.cpp)
-- 
2.33.1

